### PR TITLE
Task s4 avatar creation

### PR DESCRIPTION
**Pull request by obwansan**

### CODE REVIEW

File: css/avatar.css [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+.up {
+    transform: rotate(270deg);
+}
+
+.down {
+    transform: rotate(90deg);
+}
+
+#carColor {
+    fill: gold;
```
Comment (karimkefi) [inline css/avatar.css]: hex code here (upper case)?

File: includes/avatar.html [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    <div class="avatar down">
+        <div class="face">
+            <img src="../assets/charlie.png">
+        </div>
+        <div class="car">
+            <img src="../assets/goldCar.svg">
+        </div>
+    </div>
+
+    <!--<div class="avatar up">-->
```
Comment (karimkefi) [inline includes/avatar.html]: uncomment this block (could you also position them so that we can see both?)

File: includes/avatar.html [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    <link rel="stylesheet" href="../css/avatar.css">
+</head>
+<body>
+
+    <div class="avatar down">
+        <div class="face">
+            <img src="../assets/charlie.png">
+        </div>
+        <div class="car">
+            <img src="../assets/goldCar.svg">
```
Comment (karimkefi) [inline includes/avatar.html]: can you add an ID for charlieCar?

**Review by karimkefi — CHANGES_REQUESTED** [review]
couple of small clean ups

File: css/avatar.css (hunk 1 above)
Comment (obwansan) [reply css/avatar.css]: removed as setting colour in the svg file

File: includes/avatar.html (hunk 3 above)
Comment (obwansan) [reply includes/avatar.html]: done

File: js/avatarCreation.js [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+
+
+var playerBx = 550;
+var playerBy = 550;
+var playerBHeight = 50;
+var playerBWidth = 50;
+var playerBStartX = 550;
+var playerBStartY = 550;
+
```
Comment (nsantiagoblair) [inline js/avatarCreation.js]: You can separate `var` by commas after the first one. Group variables with the first `var`

```
var something = 'yeah',
      anodaVar = 'nope'

var anotherSetOfVars = 'here',
      init = 'fun'

File: css/avatar.css [hunk 5]
```diff
@@ -0,0 +1,33 @@
+.avatar {
+    position: relative;
+    width: 100px;
+    margin: 100px auto;
+}
+
+.face img {
+    height: 100px;
+    position: absolute;
```
Comment (nsantiagoblair) [inline css/avatar.css]: `position` before dimensions (`height`) like above.

File: includes/avatar.html [hunk 6]
```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+
```
Comment (nsantiagoblair) [inline includes/avatar.html]: whitespace

File: includes/avatar.html [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+    <title>Avatars</title>
+    <link rel="stylesheet" href="../css/avatar.css">
+</head>
+<body>
+
```
Comment (nsantiagoblair) [inline includes/avatar.html]: whitespace

File: includes/avatar.html [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+    <div class="avatar down">
+        <div class="face">
+            <img src="../assets/charlie.png">
+        </div>
+        <div class="car">
+            <img src="../assets/goldCar.svg" id="charlieCar">
+        </div>
+    </div>
+
```
Comment (nsantiagoblair) [inline includes/avatar.html]: whitespace

File: includes/avatar.html [hunk 9]
```diff
@@ -0,0 +1,30 @@
...
+    <div class="avatar down">
+        <div class="face">
+            <img src="../assets/mike.png">
+        </div>
+        <div class="car">
+            <img src="../assets/whiteCar.svg" id="mikeCar">
+        </div>
+    </div>
+
```
Comment (nsantiagoblair) [inline includes/avatar.html]: whitespace

File: js/avatarCreation.js [hunk 10]
```diff
@@ -0,0 +1,38 @@
...
```
Comment (nsantiagoblair) [inline js/avatarCreation.js]: Delete file

**Review by nsantiagoblair — COMMENTED** [review]
Looking good. Mostly whitespace

**Review by nsantiagoblair — APPROVED** [review]

**Review by karimkefi — APPROVED** [review]

**Pull request merged**